index.js

```diff
@@ -1,45 +1,57 @@
 const identity = x => x;
 const getUndefined = () => {};
 const filter = () => true;
-function createRavenMiddleware(Raven, options = {}) {
-  // TODO: Validate options.
-  const {
-    breadcrumbDataFromAction = getUndefined,
-    actionTransformer = identity,
-    stateTransformer = identity,
-    breadcrumbCategory = "redux-action",
-    filterBreadcrumbActions = filter,
-    getUserContext
-  } = options;
+const createRavenMiddleware = (function() {
+  let called = false;
+  return (Raven, options = {}) => {
+    // TODO: Validate options.
+    const {
+      breadcrumbDataFromAction = getUndefined,
+      actionTransformer = identity,
+      stateTransformer = identity,
+      breadcrumbCategory = "redux-action",
+      filterBreadcrumbActions = filter,
+      getUserContext
+    } = options;
 
-  return store => {
-    let lastAction;
-
-    Raven.setDataCallback((data, original) => {
-      const state = store.getState();
-      data.extra.lastAction = actionTransformer(lastAction);
-      data.extra.state = stateTransformer(state);
-      if (getUserContext) {
-        data.user = getUserContext(state);
+    if (!called) called = true;
+    else {
+      if (Raven._globalOptions.dataCallback instanceof Function) {
```

Comment:
I don't think this check is relevant. My intention here is that calling it twice will have a number of strange behaviors:

1. Possible memory leak.
2. Second call will replace the state/last action of the first call.
3. Breadcrumbs from both will be logged in a why that is probably indistinguishable.

Reply:
I'm guessing you added this so that you could keep your current workaround and not get the warning. I think the workaround that we identified in #50 is not a permanent solution, and you should really find a way to not include `raven-js` at all on your server (as clarified here https://github.com/captbaritone/raven-for-redux/issues/50#issuecomment-355609272)